docker/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.11.2-alpine AS builder
+
+
+WORKDIR /go/src/github.com/mteodor/edgex-app
+COPY . .
+
+RUN apk update \
+    && apk add make bash \
```

Comment:
Why do you need `bash`? AFAIK Alpine Linux uses [`ash`](https://en.wikipedia.org/wiki/Almquist_shell).

BUt never the less - why do you need shell in builder?